### PR TITLE
Add Swift/macOS sample, a port of the Windows MNist sample

### PR DESCRIPTION
**Pull request by migueldeicaza**
This is a macOS/Swift port of the MNist sample, showing how to embed the ONNX Runtime on a native Mac application and building a UI for it.

### CODE REVIEW

**Comment by migueldeicaza** [comment]
image

**Comment by snnn** [comment]
Sorry I can't take your change now. The windows one need be revised first. As you can see, the result column isn't meaningful.

File: samples/swift/SwiftMnist/OnnxInterop.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  Ort::Session session_{env, "model.onnx", Ort::SessionOptions{nullptr}};
+    
+  Ort::Value input_tensor_{nullptr};
+  std::array<int64_t, 4> input_shape_{1, 1, width_, height_};
+
+  Ort::Value output_tensor_{nullptr};
+  std::array<int64_t, 2> output_shape_{1, 10};
+};
+    
+mnist *mnist_new ()
```
Comment (snnn) [inline samples/swift/SwiftMnist/OnnxInterop.cpp:48]: Will the result get deleted at some time?

**Comment by snnn** [comment]
Thank you!

**Pull request merged**